backend/src/controllers/helperLink.controller.js

```
@@ -16,15 +16,15 @@ function validateLinks(links) {
     const { title, url, order } = link;
     if (!title || !url) {
-      result.push({ msg: "title and url are required" });
+      result.push({ msg: 'title and url are required' });
       continue;
     }
     if (!validateUrl(url)) {
-      result.push({ msg: "Invalid value for url" });
+      result.push({ msg: 'Invalid value for url' });
       continue;
     }
     if ((order && isNaN(order)) || order < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)